commands/command.go

```diff
@@ -96,7 +96,7 @@ func cmdBuilderWithInit(parent *Command, cr CmdRunner, cliText, shortdesc string
 	}
 
 	if cols := c.fmtCols; cols != nil {
-		formatHelp := fmt.Sprintf("Columns for output in a comma-separated list. Possible values: `%s`",
+		formatHelp := fmt.Sprintf("Columns for output in a comma-separated list. Possible values: ```%s`.",
```

Comment:
It looks like Cobra uses the first text within backquotes as the format for the flag.

before:

```
      --format ID            Columns for output in a comma-separated list. Possible values: ID, `Cause`, `Progress`, `Created`, `Updated`
```

after:

```
      --format               Columns for output in a comma-separated list. Possible values: `ID`, `Cause`, `Progress`, `Created`, `Updated`.
```

Reply:
Oh, this is a good trick. Might need to do this in some other places. I've always found this Cobra behavior annoying.